src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -1120,21 +1126,6 @@ public ContrastLogReader()
 
         public void Read(Context context, Stream input)
         {
-            XmlSchemaSet schemaSet = new XmlSchemaSet();
-            Assembly assembly = typeof(ContrastLogReader).Assembly;
-            var settings = new XmlReaderSettings
-            {
-                DtdProcessing = DtdProcessing.Ignore,
-                XmlResolver = null
-            };
-
-            //using (var stream = assembly.GetManifestResourceStream(ContrastLogReader.ContrastSecurityReportSchema))
-            //using (var reader = XmlReader.Create(stream, settings))
-            //{
-            //    XmlSchema schema = XmlSchema.Read(reader, new ValidationEventHandler(ReportError));
-            //    schemaSet.Add(schema);
-            //}
```

Comment:
>} [](start = 14, length = 1)

This was FxCop-specific code (the ContrastSecurity converter was based on the FxCop converter). Not needed here.